<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBox</class>
 <widget class="QDialog" name="addBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #0d0d0d;       /* deep black */
    color: #00ff66;                  /* neon green text */
    border: 1px solid #00cc44;       /* glowing green border */
    border-radius: 6px;
    padding: 6px;
    font-family: &quot;Consolas&quot;, &quot;Courier New&quot;, monospace;
    font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>Add Books</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter book:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Price:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #f1f8e9;              /* soft green input */
    border: 1px solid #66bb6a;
    border-radius: 4px;
    padding: 4px;
    color: #1b5e20;
}

QLineEdit:focus {
    border: 1px solid #2e7d32;              /* darker green when focused */
    background-color: #ffffff;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #f1f8e9;              /* soft green input */
    border: 1px solid #66bb6a;
    border-radius: 4px;
    padding: 4px;
    color: #1b5e20;
}

QLineEdit:focus {
    border: 1px solid #2e7d32;              /* darker green when focused */
    background-color: #ffffff;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #66bb6a;
    color: white;
    font-weight: bold;
    border-radius: 6px;
    padding: 6px 12px;
    border: none;
}

QPushButton:hover {
    background-color: #57a05a;
}

QPushButton:pressed {
    background-color: #388e3c;
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
